homeassistant/components/spaceapi/__init__.py

```diff
@@ -292,7 +292,7 @@ def get(self, request):
         with suppress(KeyError):
             state[ATTR_ICON] = {
                 ATTR_OPEN: spaceapi["state"][CONF_ICON_OPEN],
-                ATTR_CLOSE: spaceapi["state"][CONF_ICON_CLOSED],
+                ATTR_CLOSED: spaceapi["state"][CONF_ICON_CLOSED],
```

Comment:
This was already `spaceapi["state"][CONF_ICON_CLOSED]` so what is this actually fixing more than that you are changing the returned json?

Reply:
> so what is this actually fixing more than that you are changing the returned json

Nothing? The returned JSON is wrong, this makes it correct.

Reply:
The returned json from spaceapi is already correct. What you are changing is the presentation inside of HA which could possibly break something for someone so it's not clear to me why this change (besides the obvious fact to align the json outputs in both end).

Take into consideration I have no clue what spaceapi is or how it's used so perhaps you want to outline that so I can see why this would make sense to change.
Thanks 👍

Reply:
> The returned json from `spaceapi` is already correct.

What makes you say that? I can observe that the JSON output from the homeassitant `spaceapi` integration contains the incorrect key `state.icon.close`, rather than `state.icon.closed`.

> Take into consideration I have no clue what `spaceapi` is or how it's used so perhaps you want to outline that so I can see why this would make sense to change.

(if you don't know what it is, how can you say that the output is correct?)

[spaceapi](https://spaceapi.io/) is a JSON schema used by hackerspaces to make information about themselves available in a standardised format. The homeassistant integration provides a view that returns a JSON string that should be formatted according to this schema - however one of the keys in the returned JSON is wrong, which is what this PR is supposed to fix.

Reply:
> (if you don't know what it is, how can you say that the output is correct?)

The output from the api is correct (which you also mention). The output from HA json view is wrong which is what you're trying to fix.

Anyhow as the output changes with your fix please add the "breaking change" section in the PR description and then we can move forward.